src/main/java/wile/engineersdecor/blocks/EdCatwalkBlock.java

```diff
@@ -131,7 +131,7 @@ public List<ItemStack> dropList(BlockState state, Level world, @Nullable BlockEn
     List<ItemStack> drops = new ArrayList<>();
     drops.add(new ItemStack(state.getBlock().asItem()));
     int n = (state.getValue(NORTH)?1:0)+(state.getValue(EAST)?1:0)+(state.getValue(SOUTH)?1:0)+(state.getValue(WEST)?1:0);
-    if(n > 0) drops.add(new ItemStack(ModContent.getBlock("steel_railing"), n));
+    if(n > 0) drops.add(new ItemStack(railing_block, n));
```

Comment:
Since we already have reference to the block, might as well use it.
It's also probably worth looking into DeferredRegisters / RegistryObjects and reduce map querying across the whole mod

Reply:
Aye. I used DR/RO (and earlier object holders) for testing - for getting maintainability across mods (I like to port one or 2 to Fabric one day) quite a few wrappers and also the internal registry handling is in here, too - writing mods is a way for me to practice Java ;). For the most part the need to lookup blocks and items are seldom used and should not have noticeable performance impact.